.github/workflows/release_gem.yml

```diff
@@ -9,9 +9,13 @@ on:  # yamllint disable-line rule:truthy
 jobs:
   release:
     name: Release gem
-    uses: theforeman/actions/.github/workflows/release-gem.yml@v0
-    with:
-      allowed_owner: theforeman
-    secrets:
-      api_key: ${{ secrets.RUBYGEM_API_KEY }}
+    runs-on: ubuntu-latest
+    environment: release
+    if: github.repository_owner == 'theforeman'
+
+    permissions:
+      id-token: write
+
+    steps:
+      - uses: voxpupuli/ruby-release@v0
 ...
```

Comment:
Huh, I had to look this up

Reply:
Yes, you often see the document opening (`---`) but not often the closing line